cmd/restic/cmd_prune.go

```diff
@@ -78,6 +81,7 @@ func addPruneOptions(c *cobra.Command, pruneOptions *PruneOptions) {
 	f.BoolVar(&pruneOptions.RepackCacheableOnly, "repack-cacheable-only", false, "only repack packs which are cacheable")
 	f.BoolVar(&pruneOptions.RepackSmall, "repack-small", false, "repack pack files below 80% of target pack size")
 	f.BoolVar(&pruneOptions.RepackUncompressed, "repack-uncompressed", false, "repack all uncompressed data")
+	f.StringVar(&pruneOptions.SmallPackSize, "small-pack-size", "", "pack `below-limit` packfiles smaller than (allowed suffixes: k/K, m/M, g/G, t/T)")
```

Comment:
I just noticed that the original issue suggested `--repack-smaller-than` as option. Is there a particular reason to deviate from that?

P.S. the description should be  (pack -> repack, packfiles -> pack files)
```
"repack pack files smaller than `below-limit` (allowed suffixes: k/K, m/M, g/G, t/T)"
```
We could also drop the `t/T` suffix as it will always lead to an error. `g/G` currently is also always an error as the pack size is still limited to 128MB, but that could change in the future.